jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/UpgradeRequest.java

```diff
@@ -31,26 +31,6 @@
  */
 public interface UpgradeRequest
 {
-    /**
-     * Add WebSocket Extension Configuration(s) to Upgrade Request.
-     * <p>
-     * This is merely the list of requested Extensions to use, see {@link UpgradeResponse#getExtensions()} for what was
-     * negotiated
-     *
-     * @param configs the configuration(s) to add
-     */
-    void addExtensions(ExtensionConfig... configs);
-
-    /**
-     * Add WebSocket Extension Configuration(s) to request
-     * <p>
-     * This is merely the list of requested Extensions to use, see {@link UpgradeResponse#getExtensions()} for what was
-     * negotiated
-     *
-     * @param configs the configuration(s) to add
-     */
-    void addExtensions(String... configs);
-
```

Comment:
The changes in this file specifically bother me, but I cannot articulate why (yet).